sagafalabella/sagafalabella/spiders/falabella.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from urllib.parse import quote_plus
+import json
+from scrapy import Request
+
+class FalabellaSpider(scrapy.Spider):
+    name = 'falabella'
+    allowed_domains = ['falabella.com.pe/falabella-pe']
```

Comment:
Why not `falabella.com.pe`

Reply:
Because `falabella.com.pe` redirects to `falabella.com.pe/falabella-pe`